test/clojure/nrepl/middleware/lookup_test.clj

```diff
@@ -15,6 +15,7 @@
 
 (def-repl-test lookup-op
   (doseq [op [{:op "lookup" :sym "map" :ns "clojure.core"}
+              {:op "lookup" :sym "let" :ns "clojure.core"}
```

Comment:
I think it's going to be nice to update the unit test as well, not just the integration test.

Reply:
True, done. I initially figured it was only really the bencoding part that was the problem, but it's definitely good to have the unit test as well.